chromadb/db/index/hnswlib.py

```diff
@@ -250,11 +252,20 @@ def get_nearest_neighbors(self, query, k, ids=None):
         if len(labels) != 0:
             filter_function = lambda label: label in labels
 
+        preprocessed_query = query
+        # HNSW does not normalize the query when using cosine distance. This causes incorrect distances
```

Comment:
Is this correct?

https://github.com/nmslib/hnswlib/blob/359b2ba87358224963986f709e593d799064ace6/python_bindings/bindings.cpp#L630

Reply:
You're right.